tests/main/snap-validate-basic/vs1.assert

```diff
@@ -0,0 +1,29 @@
+type: validation-set
+authority-id: xSfWKGdLoQBoQx88vIM1MpbFNMq53t1f
+series: 16
+account-id: xSfWKGdLoQBoQx88vIM1MpbFNMq53t1f
+name: bar
+sequence: 1
+snaps:
```

Comment:
Could we add a snap with optional presence?

Reply:
Yep, I can add it, just found it mildly interesting for the test ;)

Reply:
Added